contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -55,8 +55,19 @@ def run(self, packs, abs_repo_base, verifyssl=True, force=False):
             pack_url, pack_version = self._get_repo_url(pack)
 
             temp_dir = hashlib.md5(pack_url).hexdigest()
+            lock_file = LockFile(temp_dir)
```

Comment:
I think It will try to create `/12312.lock` in system root dir,
while before it was `/tmp/12323.lock`